vmwc/__init__.py

```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
         if not resp.status_code == 200:
             raise Exception("Error while uploading file")
 
+    def download_file(self, guest_path, local_path):
+        if self._tools_credentials is None:
+            raise Exception("Login required to use guest functions")
+
+        fti = self._client.get_guest_operations_manager().fileManager.InitiateFileTransferFromGuest(vm=self._raw_virtual_machine,
+                                                                                                    auth=self._tools_credentials,
+                                                                                                    guestFilePath=guest_path)
+
+        resp = requests.get(fti.url.replace("https://*", "https://" + self._client.host), verify=False)
```

Comment:
you can use `self._client.verify` to determine if SSL verification is required in this session